fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogBottomSheet.kt

```diff
@@ -632,12 +632,12 @@ private fun TranslationsDropdown(
                     offset = if (isInLandscapeMode) {
                         DpOffset(
                             -contextMenuWidthDp + ICON_SIZE,
-                            -ICON_SIZE,
+                            ICON_SIZE,
```

Comment:
Self note: from Mihai's commit message, this change landed in the [`1.6.0-alpha04`](https://developer.android.com/jetpack/androidx/releases/compose-material?hl=en#1.6.0-alpha04):

> Fixed `DropdownMenu`'s `offset` calculation so x offsets depend solely on the local layout direction, and y offsets will no longer be reversed when the menu is near the bottom of the screen.

Reply:
Thanks for fixing this during the upgrade, and in a separate commit so it's clear why it's here and also doesn't include a regression!